e2e/bitrot_test.go

```diff
@@ -14,20 +14,20 @@ func TestBitrot(t *testing.T) {
 
 	r := require.New(t)
 
-	gds, err = setupCluster("./config/1.toml", "./config/2.toml")
+	tc, err := setupCluster("./config/1.toml", "./config/2.toml")
 	r.Nil(err)
-	defer teardownCluster(gds)
+	defer teardownCluster(tc)
 
-	client = initRestclient(gds[0])
+	client = initRestclient(tc.gds[0])
 
 	// test Bitrot on dist-rep volume
-	t.Run("Replica-volume", testBitrotOnReplicaVolume)
+	t.Run("Replica-volume", func(t *testing.T) { testBitrotOnReplicaVolume(t, tc) })
```

Comment:
Suggestion: you can write a wrapper function if this pattern repeats. In pseudo-Go code:

```go
wrapTest(t, testBitrotOnReplicaVolume, tc)
```

Reply:
I had thought about that and decided to defer doing that. Since you've suggested it I will add it.